api/src/models/team.model.js

```diff
@@ -55,6 +55,11 @@ const teamSchema = new mongoose.Schema(
62vrbypCeRrQggbClCSB2nphflcPqf/8V/m6zyAwDaMjuETIfNawgfcaXxi+ViG9jSjnffdcWZ2ZaWVyOIfLuOh+qf/dN/Uvy//+N/zvhbDiUEgHps/lnvz//1/1H83//PC8XZ12q5OHi+KIoDzvhgHgIIcytDSJRUb672qM37mhNCAKC6zd93P/HxjxW/eOtKHV/Aml2SCCAkWVpejQ1ZJ6qFkJTXnBACANVs9b5by3vss0VRHBI+SCGAkGxpefX2oiiOFkVxb96jJ4AAQLs2e++t/B77rfW1lcc8kaQyhE6yDWt6n/XoAcAofU34IJcKCJUsLa/GpouDaf8NFRAAaE+tLViXy2HzE55AcqmAUEm5pvdrHkUAGLzpsLnwQSUqINQibThdBQQA2lNLBeRkWfkwbE5lAgi1mX84XQABgHZs954793usYXNqpQWL2mwYTq98cvr7BGQAWJDLhs1pggoIjVhaXn2iKIrHN/9vp73mPvjBDxa/+c1vPVEAkCy7AuJkcxojgNCYpeXVA0VRHL5xLiT1NacNCwDSzXq/3fL91bwHjRJAaNTmcyECCAA0LyuAmPegcWZAaNT/3979q8ZxRQEcHlSFkIAJ6a03sN/AegOp29Kqto0C6qP0C5FbVetSnfQG2jeQ3kDbhyCREFKZMHAGrmTr32r3zp073wciqmeMzS/n3LmbORcCADzuxf+D2XkPsjEBIZvJdNb+pfaHJw4ARWnv99h33oNcBAhZvey+EABgw84jPpz3IBsBQnaT6exNnAv54OkDQG9+PT05PPb4yU2A0JvHP9ULAGyIT+zSKwFCrybT2U5MQ6xkAcDmWbmidwKE3lnJAoAsrFxRBAFCMaxkAcBGWLmiKAKEosRXstppyFtvBgBezcoVxREgFCdWsuZN0+x6OwCwkvZiwYPTk8O5x0dpBAjFmkxn+03THDugDgAv4mJBiiZAKNpkOtuOlax33hQAPOnT6cnhgcdEyQQIg+CAOgA8ahlTjwuPidIJEAYj7gyZO6AOAHc4aM6gCBAGJQ6ot9OQX7w5AEbuNsLjbOwPgmERIAzSZDrbi2mIA+oAjNEi7vYw9WBwBAiD5XO9AIxQO/U4cqM5QyZAGDzTEABGYhErV9deOEMmQKiCaQgAFTP1oCoChKqYhgBQGVMPqiNAqI5pCAAVMPWgWgKEark3BICBcq8HVRMgVM29IQAMiHs9GAUBwijENKQdY7/zxgEo0KdYuTL1oHoChFGZTGftNOTAIXUACnHV/rt0enJ44YUwFgKE0ZlMZ9txNuSDtw9AT9p1q+PTk8MjL4CxESCMVnyy99ghdQAyO4+ph0/rMkoChFGLQ+rtStZvY38WAGzcMsLDIXNGTYCAtSwANu/3WLlyyJzREyCQsJYFwJpZt4J7BAjck6xl+VoWAKtaxp0evm4F9wgQeECsZbXTkF3PCIBn8nUreIIAgSe4xBCAZ/oc61bOecAjBAg802Q6248QsZYFQGoR4XHpqcDTBAi8gPMhACSc84AVCBBYQZwPafd7P3p+AKNzGxOPuVcPLydA4BXcHwIwKrexius+D3gFAQJrEAfVj4QIQLUcMIc1ESCwRi4yBKhOGx5HLhKE9REgsAHxxawjIQIwWIs4YC48YM0ECGxQ7hD5+acfmz//+tsrBVjdIiYevmwFGyJAIIPJdHaU89O9W1tbzZcvX7xagOcTHpCJAIFM3CECUCThAZkJEMhMiAAUQXhATwQI9CRCZM9hdYCshAf0TIBAAXw1C2DjhAcUQoBAQfoKkR++/67559///FEAauQeDyiMAIECxYWGB25WB1iZ8IBCCRAo2GQ624kQ2fWeAJ502zTNcftzenJ443FBmQQIDMBkOtuO1ayP3hfAV5bxd+SZ8IDyCRAYEJ/wBbijPVg+Pz05nHssMBwCBAYqDqy3IfLOOwRG5nOsWV168TA8AgQGLs6J7FvPAirXne+YO1gOwyZAoBJxTmTfehZQmauYdlizgkoIEKhQrGft+4wvMFDttOPMmhXUSYBAxSbT2fuYiOyZigADsEzWrHzNCiolQGAE4utZew6tA4X6HNFx4QVB/QQIjIypCFCI9mzH3LQDxkeAwIg5KwJk5mwHIECAO1/Qan/eeiTAmp3HLeW+ZAUIEOAuK1rAmnQrVmfu7QBSAgR40GQ624upyK6nBDzDMlas5lasgIcIEOBJyVe09sQIcM9tEh2+YgU8SYAALxIx0p0X8UlfGKcuOtr1qjN/BoCXECDAyuLw+p4YgVEQHcBaCBBgLSJGdqxpQVVEB7B2AgRYu3tnRnZ8TQsGZZlEhzMdwNoJEGDj4mtaXYy4ZwTK030y98LXq4BNEyBAVnHPSBckzo1AP9rVqotk0nHjPQC5CBCgN8mq1o6LD2HjrrrosFoF9EmAAMVIpiNtkHzwZuBVugPkF7Fa5TZyoAgCBChSTEd2kh/rWvC08yQ4nOUAiiRAgEEQJPBNiyQ4rFUBgyBAgEESJIxQu1J1KTiAoRMgQBWSIHnvDAmVWN4LDitVQBUECFCtONSeRok7SCjZIgmOS4fGgVoJEGA0YkqSRsl7UUJPriI2Lk03gLERIMCoiRIy6CYb1zHZcHYDGDUBAvANk+msC5LtJExclMhjlhEZF11wmGwAfE2AADxTMi0RJuPWrk/dRGhcR2iYagA8kwABWIOYmGwnYfLGl7gGrZtmXCaxcWOiAfB6AgRgg5Kpybf+6+6S/nR3ajRdXHSxITIANkuAAPQspidNEijdJKWx4rWSbnrRRFw03aqUwADonwABGIBkktIk05TOTvL7doVf8Vokv98kk4s7vwsLgGEQIAAVuxcuqW7a8pB0CrOq7vzEQ66TSUVHSABUToAAAADZbHnUAABALgIEAADIRoAAAADZCBAAACAbAQIAAGQjQAAAgGwECAAAkI0AAQAAshEgAABANgIEAADIRoAAAADZCBAAACCPpmn+B8WelWeVKuDHAAAAAElFTkSuQmCC',
```

Comment:
Hmm it would be a bit hard to do since we'll have to manually update the front end for **each** of the .js pages that uses team member image (both scholly layouts and client). A rough solution to prevent redundant if-statements would be to put "default" in the model.

Unless someone might have a suggestion for this?